src/features/key-stats/components/KeyStats.tsx

```diff
@@ -2,13 +2,24 @@ import React from "react";
 import { useSelector } from "react-redux";
 import { Loading } from "../../loading";
 import "./KeyStats.css";
+import { AppState } from "../../../store";
+import { KeyStatsData } from "../redux/actions";
+import { ErrorMessage } from "../../error-message";
 
 const NUMBER_FORMATTER = new Intl.NumberFormat();
 
-const DEFAULT_FORMATTER = (data, key) =>
-  data[key] != null ? data[key] : "N/A";
+const DEFAULT_FORMATTER = (data: KeyStatsData, key: keyof KeyStatsData) =>
+  data[key] != null ? String(data[key]) : "N/A";
```

Comment:
This is a very nice use of `keyof` here to pick items of out of `KeyStatsData` 👍 

I think `data[key] != null` should be `!data[key]`. One thing to watch is to use the `!==`  or `===` operators for comparisons which don't try to coalesce the types

Using the `String()` global object could be a problem as the contents of `data[key]` could be anything. You might want to check that it's a string or a number.